woocommerce/woocommerce-ios
# [Mobile Payments] Launch the feature, ignore the beta flag

Closes #5028 

Changes:
- Ignores the `.isCardPresentEligible` beta flag in the account response.

Note:
- The learn more attributed text was causing a crash for me consistently. I've reduced it to a simpler string for now and opened #5031 to fix it properly. This PR can proceed.

To test:

US Store
- Set up a US store with WooCommerce Payments (dev tools is fine)
- Select that store in the app
- Ensure you can navigate to Settings (gear) > In-Person Payments > Order card reader (will load 404 in webview for now)
- Ensure you can navigate to Settings (gear) > In-Person Payments > Manage card reader
- Ensure you can navigate to Settings (gear) > Help & Support > Contact WooCommerce Payments Support

US Store w/o WCPay active
- Deactivate the WooCommerce Payments plugin
- Navigate to Settings (gear) > In-Person Payments
- Ensure you are prompted to activate the plugin
- Ensure you can navigate to Settings (gear) > Help & Support > Contact WooCommerce Payments Support
- ...

US Store w/ WCPay not installed
- Uninstall the WooCommerce Payments plugin
- Navigate to Settings (gear) > In-Person Payments
- Ensure you are prompted to install the plugin
- Ensure you can navigate to Settings (gear) > Help & Support > Contact WooCommerce Payments Support
- ...

Canadian Store
- Set up a CA store with WooCommerce Payments (dev tools is fine) (??? Is it sufficient to change the store's base location, or is a whole separate WCPay CA account needed too?)
- Select that store in the app
- Navigate to Settings (gear) > In-Person Payments
- Ensure you are notified that In-Person Payments is not yet available in Canada
- Ensure you can navigate to Settings (gear) > Help & Support > Contact WooCommerce Payments Support
- ...

Update release notes:
